lib/grape-active_model_serializers/formatter.rb

```diff
@@ -26,21 +26,21 @@ def fetch_serializer(resource, env)
 
         def other_options
           options = {}
-          if @meta_content_items
-            if @meta_key
-             key_option = @meta_key[:meta_key]
-              @meta_key.delete(:meta_key)
+          if Formatter::ActiveModelSerializers.meta
+            if Formatter::ActiveModelSerializers.meta_key
+              key_option = Formatter::ActiveModelSerializers.meta_key[:meta_key]
+              Formatter::ActiveModelSerializers.meta_key.delete(:meta_key)
```

Comment:
we can just do `key_option = Formatter::ActiveModelSerializers.meta_key.delete(:meta_key)` here.